src/frontend/js/revisions.ts

```diff
@@ -56,7 +56,7 @@ function handleRevisionSliderInput({ target }: Event) {
     .getAttribute("data-date");
   // Update the revision info box
   revisionInfo.textContent =
-  `Revision: ${currentRevision}\r\n${revisionDate}`;
+  `Verision: ${currentRevision}\r\n${revisionDate}`;
   // Calculate position of revision info box to make sure it stays within the area of the slider position
   revisionInfo.style.left = `calc(${position}% + (${125 - position * 2.5}px))`;
```

Comment:
```suggestion
  revisionInfo.style.left = `calc(${position}% + (${revisionInfo.offsetWidth / 2 - position * revisionInfo.offsetWidth / 100}px))`;
```
Because the editor name can be wider than the available space, I suggest to use a variable width here.
Since the width before was `250px` and the "magic" values used in the calculations were `125` and `2.5`, they have to be replaced with the actual `offsetWidth / 2` and `offsetWidth / 100`.
Also, this means that we would have to delete the lines https://github.com/Integreat/integreat-cms/blob/e831704f508106afab01abb6a31af270415e6ea5/src/frontend/css/style.scss#L303 and  https://github.com/Integreat/integreat-cms/blob/e831704f508106afab01abb6a31af270415e6ea5/src/frontend/css/style.scss#L306 in the stylesheet.